cfme/tests/containers/test_one_image_analysis.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from utils import testgen
+from utils.version import current_version
+from cfme.web_ui import flash, toolbar as tb, tabstrip as tabs
+from cfme.containers import image
+from utils.wait import wait_for, TimedOutError
+from cfme.configure import tasks
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+
+# CMP-10064
+
+def test_image_analysis_finished():
+    """ This module verifies the SSA scan
+        On the Images page ---> put a checkmark next to the image
+        Click on Configuration ---> Perform SmartState Analysis
+        Verify the relevant notification is displayed
+        Settings ---> Tasks ---> All VM and Container Analysis Tasks
+        Verify the image is being scanned
+        Wait for the scan to finish ---> Delete All
+    """
+    sel.force_navigate('containers_images')
+    m = 'Analysis successfully initiated'
+    sel.check(image.checkbox)
```

Comment:
This shouldn't be here, but rather a method on the ```Image``` class should be created called ```run_smartstate_analysis```. In this way others can benefit from this too. The ```force_navigate``` at the beginning of the test and lines 35-40 should be part of this new method too. This test then becomes two steps:

```
image.run_smartstate_analysis()
is_task_finished()
```